src/Language/Printer.php

```diff
@@ -226,6 +228,7 @@ public function printAST($ast)
                     },
 
                     NodeKind::NAMED_TYPE => static function (NamedTypeNode $node) : string {
+                        // @phpstan-ignore-next-line the printer works bottom up, so this is already a string here
```

Comment:
Why does PHPStan not respect that ignore?

Reply:
Not available until `0.12.18`, and we're on ` 0.12.18`..?

Reply:
I see, guess i will attempt an upgrade then.

Reply:
Upgrade done.